### DISCUSSION TITLE
Is "https://sqlitestudio.pl/" down or dead?

### DISCUSSION THREAD

**Discussion by psdevil** [pawelsalawa/sqlitestudio#4506] [General]
I was trying to see if there was a newer version (I'm using v3.2.1) but I was unable to get to the site...

![image](https://user-images.githubusercontent.com/60613594/188947448-7eb6622b-e53f-4b5b-b568-25aa3e458d00.png)

**Comment by pawelsalawa** [comment]
Thanks for pointing this out. Fixed.

**Comment by kenferrigni** [comment]
It's down again, I'm afraid.

**Reply by pawelsalawa** [reply]
Thanks. I've fixed it again. I'm not sure what's causing it. Logs are inconclusive :(

**Reply by kenferrigni** [reply]
Your response time is nothing short of heroic. Thank you!!

**Reply by pawelsalawa** [reply]
It just happened I was going through the mailbox, when the github notification arrived :)
